Essentials/src/main/java/com/earth2me/essentials/RandomTeleport.java

```diff
@@ -195,8 +195,8 @@ public void cacheRandomLocations(final Location center, final double minRange, f
 
     // Returns an appropriate elevation for a given location in the nether, or -1 if none is found
```

Comment:
A value of -1 was used as a sentinel value here to indicate that no safe height was found in the nether. Probably best change this now since a negative world height in the nether is possible. Will need a change in `isValidRandomLocation` for this as well (which checks for a valid height value). This method also starts at height y = 32 which is where the lava surface normally lies. In prior versions below this level there was normally no safe location, so it was skipped. Not sure if this assumption holds still.
```suggestion
    // Returns an appropriate elevation for a given location in the nether, or Double.MIN_VALUE if none is found
```